api/.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+	root: true,
+	env: {
+		node: true,
+	},
+	extends: [
+		'plugin:@typescript-eslint/recommended',
+		// 'plugin:prettier/recommended'
+		// Uncomment when we are ready to prettify everything
```

Comment:
Lets wrap up this PR with the configs, and then do 1 more PR auto-fixing everything based on the new rules 👍